### PR TITLE
DEVDOCS-1999: Apps - Prod | Building Apps 12: Approval Requirements

### PR DESCRIPTION
**Pull request by aglensmith**
# [DEVDOCS-1999](https://jira.bigcommerce.com/browse/DEVDOCS-1999)

## What changed?
* added approval requirements.

### CODE REVIEW

File: docs/api-docs/apps/guide/apps-12-requirements.md [hunk 1]
```diff
@@ -2,63 +2,74 @@
...
+- Apps must serve all callback URLs over HTTPS.
+- Apps that process transactions or handle credit card data must pass a PCI Compliance review by BigCommerce's security team.
 
 ## Installation
-- Apps should be Single-Click and use our OAuth token exchange flow to authenticate the user
-- Single-Click does not mean everything  completed in one click, but that authentication is handled in that click - other tasks can be handled post-auth with the user
-- When the app is installed, a styled HTML page must be returned in the iframe of the BigCommerce control panel - this content cannot be blank and must follow our [user-interface constraints](https://developer.bigcommerce.com/api-docs/getting-started/building-apps-bigcommerce/building-apps#building-apps_user-interface-constraints).
-- During the OAuth handshake, be sure to store the user’s token against their store hash - rather than their email address - to avoid any problems with ownership changes or multi-user support.
+- Apps should be [single-click](https://developer.bigcommerce.com/api-docs/apps/guide/types-of-apps#single-click-apps) and use OAuth flow to authenticate<sup>1</sup>.
```
Comment (mikrdgz) [inline docs/api-docs/apps/guide/apps-12-requirements.md:33]:
```suggestion
- Apps should be [single-click](https://developer.bigcommerce.com/api-docs/apps/guide/types-of-apps#single-click-apps) and use OAuth flow to authenticate.

```

File: docs/api-docs/apps/guide/apps-12-requirements.md [hunk 2]
```diff
@@ -2,63 +2,74 @@
...
+- Apps must store user tokens against the store hash, not the user's email address, to avoid problems with ownership changes and multiple user support.
 
 <div class="HubBlock--callout">
 <div class="CalloutBlock--info">
 <div class="HubBlock-content">
 
 <!-- theme:  -->
 ### Note
-> * If your app cannot meet these requirements, please reach out to <a href="mailto:appstore@bigcommerce.com">appstore@bigcommerce.com</a> in order to discuss the potential for a Connector app listing. In the majority of cases, a Single-Click app will be the more appropriate solution and put your integration on par with other Marketplace partners, but exceptions may be made under certain circumstances
+> 1. If your app cannot meet this requirement, email <a href="mailto:appstore@bigcommerce.com">appstore@bigcommerce.com</a> to discuss the potential for a connector app listing.
```
Comment (mikrdgz) [inline docs/api-docs/apps/guide/apps-12-requirements.md:43]:
```suggestion
> If your app cannot meet this requirement, email <a href="mailto:appstore@bigcommerce.com">appstore@bigcommerce.com</a> to discuss the potential for a connector app listing.

```

Comment (aglensmith) [reply docs/api-docs/apps/guide/apps-12-requirements.md:43]: Nope. That's there for a reason.

File: docs/api-docs/apps/guide/apps-12-requirements.md [hunk 3]
```diff
@@ -2,63 +2,74 @@
...
-- Utilize any V3 API resources that are currently available for the data you need from a BigCommerce store.
-- The dashboard and callback URLs for your app should operate strictly over HTTPS using a valid SSL certificate.
-- Apps should offer [multi-user support](https://developer.bigcommerce.com/api-docs/getting-started/building-apps-bigcommerce/building-apps#building-apps_multi-user-support) so that permissions for the app can be distributed by the store owner to additional users, from the BigCommerce control panel.
-- Any apps that process transactions or handle credit card data will need to pass a PCI Compliance review with our Security team, and also be compatible with our Optimized One-Page Checkout.
-- Avoid copy/paste for storefront code - use the [Scripts API](https://developer.bigcommerce.com/api-reference/storefront/content-scripts-api) to programmatically inject code snippets into a store’s Script Manager and storefront theme.
-- If your app requires payment, you’ll need to collect payment from the user within your app. See Pricing & Categorization under App Summary for more information.
+- Apps must work as intended and can not conflict with BigCommerce functionality
```
Comment (uwikaiddi) [inline docs/api-docs/apps/guide/apps-12-requirements.md:27]:
```suggestion
- Apps must work as intended and can not conflict with BigCommerce functionality.

```

File: docs/api-docs/apps/guide/apps-12-requirements.md [hunk 4]
```diff
@@ -2,63 +2,75 @@
...
-## General Requirements
-- Make Support resources available throughout the app experience (for example onboarding flow and app dashboard).
-- Follow BigCommerce branding guidelines when referencing BigCommerce (see our [Media Kit](https://www.bigcommerce.com/press/media-kit/) for more information).
-- Apps and listing must not reference competitor platforms.
-- Brand name used in App submission should match the app's title used in app content.
-- Keep app titles concise -- refrain from using extra taglines or descriptors (include those in the app summary, if necessary).
-- Apps that connect to third-party service should include the name of the service and the company in the title (Ex: "Marketplace by BigCommerce", "Search by BigCommerce").
-- App client-side code (including any storefront components) should be compatible with all [BigCommerce supported browsers](https://support.bigcommerce.com/s/article/Themes-Supported-Browsers).
+## General requirements
+- Make support resources available throughout the app (for example, during onboarding and on the app's dashboard).
```
Comment (uwikaiddi) [inline docs/api-docs/apps/guide/apps-12-requirements.md:18]:
```suggestion
- Make support resources available throughout the app, like during onboarding, and on the app's dashboard.

```

File: docs/api-docs/apps/guide/apps-12-requirements.md [hunk 5]
```diff
@@ -2,63 +2,75 @@
...
+- Follow branding guidelines when referencing BigCommerce (see our [Media Kit](https://www.bigcommerce.com/press/media-kit/) for more information).
+- Don't reference competitor platforms in the app's listing information.
+- Use the same app name in app submission and in app content.
+- Keep app titles concise. Avoid using extra taglines or descriptors; include those in the app summary, if necessary.
+- Include the name of connected third-party platforms in the title (ex: "Marketplace by BigCommerce" or "Search by BigCommerce").
```
Comment (uwikaiddi) [inline docs/api-docs/apps/guide/apps-12-requirements.md:23]:
```suggestion
- Include the name of connected third-party platforms in the title (For example: "Marketplace by BigCommerce" or "Search by BigCommerce").

```

**Pull request merged**